coreneuron/apps/main1.cpp

```diff
@@ -467,6 +467,8 @@ static void* load_dynamic_mpi(const std::string& libname) {
 #endif
 
 extern "C" void mk_mech_init(int argc, char** argv) {
+    corenrn_param.multisend = false;
+    corenrn_param.binqueue = false;
```

Comment:
Originally hoping for something like ```corenrn_param = corenrn_parameters()``` but that is not allowed and I didn't want to convert to a pointer.